---
layout: paper
title: "Assessing Long-Term Stored Tissues for Multi-Omics Data Quality and Proteogenomics Suitability"
image: ""
authors: "Kyu Jin Song, Minsuh Kim, Yong Jin Heo, Kyung-Cho Cho, Jae-Won Oh, Dae Ho Kim, Chanwoong Hwa, Yeji Do, Seunghyuk Choi, Hee Sang Hwang, Kwoneel Kim, Kyunggon Kim, Seungjin Na, Eunok Paek, Joon-Yong An, Se Jin Jang, Min-Sik Kim, Kwang Pyo Kim."
year: 2025
ref: "Song et al., Journal of Proteome Research."
journal: "Journal of Proteome Research"
doi: "10.1021/acs.jproteome.5c00289"
---

# Abstract

As research into cancer biology progresses, multiomics analyses have become essential for unraveling its molecular complexities. However, sample availability remains a challenge due to factors such as collection procedures and long-term storage effects. Archived samples present an opportunity to expand multiomics studies, but concerns persist regarding storage duration's impact on data reliability. This study examines the genomic, transcriptomic, and proteomic profiles of samples stored for over a decade. Transcriptomic analysis revealed a decline in read counts for protein-coding genes but preserved core gene expression patterns. Proteomic measurements remained stable, with minimal changes in post-translational modifications. While phosphorylation and acetylation rates were largely unaffected, a slight increase in modification frequencies was observed. Housekeeping genes and proteins exhibited consistent expression across samples, yet proteomic differences between the tumor and normal tissues were distinct. Despite technical variations in transcriptomic data, essential transcription factors and kinases retained functionality. These findings underscore the viability of archived samples for multiomics research, enabling broader investigations into cancer biology and providing insights into molecular mechanisms. By leveraging archived specimens, researchers can overcome sample limitations and advance precision oncology efforts, ultimately deepening our understanding of cancer at the systems level.
